### ISSUE TITLE
pagefileSecure and pagefileExtendedPaths and user image file. "notice" causes "header already sent", thus not loading the image

### ISSUE THREAD

**Issue by somatonic** [processwire/processwire-issues#688]
### Short description of the issue

Setup: pagefileSecure and pagefileExtendedPaths is on. User template has a image field.

Users have view access to user template, so the image can be rendered. It all works fine but there's some bad coding...

Now when rendering a profile on the site (frontend) outputting the image and if it's the **current** user viewing his own profile the image isn't loaded (header already sent) in debug mode, cause there's a notice happening in PagePermissions.module ~#245 cause $this->wire("page") seems to be null... so there's no method ->process

```
// if user is editing themselves in ProcessProfile, and they have permission to do so
if($user->id === $page->id && $this->wire('page')->process == 'ProcessProfile' && $user->hasPermission('profile-edit')) return true;
```

* ProcessWire version: 3.0.98

**Comment by netcarver** [comment]
@somatonic Is this a duplicate of #629 - which was fixed around ~3.0.106 (IIRC)

**Comment by ryancramerdesign** [comment]
Closing because it appears to be a duplicate (per netcarver) and should be already fixed, though let me know if still seeing the issue there.
